src/navigation/sim_translator/sim_translator/node_cone_detection_translator.py

```diff
@@ -0,0 +1,79 @@
+from math import sin, cos, sqrt
+import rclpy
+from rclpy.node import Node
+from rclpy.publisher import Publisher
+from nav_msgs.msg import Odometry
+from geometry_msgs.msg import Point
+from visualization_msgs.msg import MarkerArray
+from driverless_msgs.msg import ConeDetectionStamped, Cone
+from fs_msgs.msg import Track, Cone as FSCone
+from driverless_common.marker import marker_array_from_cone_detection
+
+from transforms3d.euler import quat2euler
+
+from typing import List
+
+
+MAX_CONE_RANGE = 10  # meters
+
+
+class ConeDetectonTranslator(Node):
+    track: List[FSCone] = []
+
+    def __init__(self) -> None:
+        super().__init__("cone_detection_translator")
+        self.create_subscription(Track, "/testing_only/track", self.track_callback, 10)
+        self.create_subscription(Odometry, "/testing_only/odom", self.odom_callback, 10)
+
+        self.detection_publisher: Publisher = self.create_publisher(ConeDetectionStamped, "cone_detection", 1)
```

Comment:
we've got a few other `cone_detection` topics now, maybe `/translator/cone_detection` to ease confusion?